bench-vortex/src/statpopgen/schema.rs

```diff
@@ -0,0 +1,1206 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: Copyright the Vortex contributors
+
+use std::sync::{Arc, LazyLock};
+
+use arrow_schema::DataType::*;
+use arrow_schema::{DataType, Field, Schema, SchemaRef};
+
+pub fn list(x: DataType) -> DataType {
+    List(Arc::new(Field::new("item", x, true)))
+}
+
+pub static SCHEMA: LazyLock<SchemaRef> = LazyLock::new(|| {
```

Comment:
Given how extensive the schema is, could we generate the dynamically from the vcf header?

Reply:
Sure. I need to fix this benchmark to be faster at downloading the data anyway. I'll do those things together at some point soon. Until then I don't want to merge.

Reply:
This still seems tedious / error prone to spell out by hand. Your call. 🙂